helm/crunchy-monitoring/templates/secrets/grafanaSecret.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+data:
+  password: YWRtaW4=
+  username: YWRtaW4=
```

Comment:
Those should probably be randomly generated on install, and use a helm lookup on subsequent updates to avoid re-generating them